src/LowLevelMotionController.cpp

```diff
@@ -136,6 +142,16 @@ int main(int argc, char **argv)
 
     while (ros::ok())
     {
+        // Check the safety client before updating anything
+        ROS_ASSERT_MSG(!safety_client.isFatalActive(), "low_level_motion: fatal event from safety");
+
+        if(safety_client.isSafetyActive())
+        {
+            // Override whatever the Acceleration Planner wants to do and attempt to land
+            geometry_msgs::Twist twist;
+            twist.linear.z = -0.2; // Try to descend
+            quadController.setTargetVelocity(twist);
```

Comment:
Don't we also need to tell the quadController to not accept new requested velocities?